swaggertooling/openapi-pipeline/src/main/java/uk/gov/ch/pipe/NaiveConverterPipe.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.ch.pipe;
+
+import uk.gov.ch.swagger.VersionConverter;
+
+import java.nio.file.Path;
+
+public class NaiveConverterPipe extends AbstractAPIPipe {
+    @Override
+    protected void handle() {
+        try {
+            convert(getArgs().getFixedDir(),
+                    getArgs().getConvertDir()
+            );
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Please log the exception, not `printStackTrace()`

Reply:
Created a static method to pass the logger from the AbstractAPIPipe here, so that this class isn't effectively using two different loggers.
Right thing to do or not?